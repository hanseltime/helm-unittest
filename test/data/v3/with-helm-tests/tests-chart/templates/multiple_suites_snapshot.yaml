{{- range $idx, $env := $.Values.envs }}
---
{{- if $.Values.includeSuite }}
suite: multiple test suites snapshot {{ $env }}
{{- end }}
{{- if $.Values.customSnapshotIds }}
snapshotId: {{ $env }}
{{- end }}
templates:
  - templates/service.yaml
{{- if $.Values.unexpectedField }}
something: here
{{- end}}
tests:
  - it: manifest should match snapshot
    set:
      env: {{ $env }}
    asserts:
      - matchSnapshot: {}
{{- end }}