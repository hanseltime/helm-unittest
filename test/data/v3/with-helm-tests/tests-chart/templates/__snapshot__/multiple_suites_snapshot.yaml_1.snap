manifest should match snapshot:
  1: |
    apiVersion: v1
    kind: Service
    metadata:
      labels:
        app: with-subchart
        chart: with-subchart-0.1.0
        heritage: Helm
        release: RELEASE-NAME
      name: RELEASE-NAME-with-subchart
    spec:
      ports:
        - name: nginx
          port: 80
          protocol: TCP
          targetPort: 80
      selector:
        app: with-subchart
        env: prod
        release: RELEASE-NAME
      type: ClusterIP
